tests/functional/autoprompt/test_prompttoolkit.py

```diff
@@ -106,7 +106,8 @@ def setUp(self):
             driver=self.driver,
             factory=self.factory,
             cli_parser=self.cli_parser,
-            output=FakeApplicationOutput()
+            output=FakeApplicationOutput(),
+            app_input=FakeApplicationInput(),
```

Comment:
Not recognizing this change either, is this testing infra specific to v2?

Reply:
Yeah these are v2 changes. I put all of the v2 changes in a separate commit (the second one).